search.py

```diff
@@ -420,8 +420,35 @@ def astar_search(problem, h=None, display=False):
     return best_first_graph_search(problem, lambda n: n.path_cost + h(n), display)
 
 
+def cost_limited_astar_search(problem, limit, f):
+
+    def recursive_clas(node, problem, limit, f):
```

Comment:
Is that a type? What is 'clas'?

Reply:
I changed the function name to "recursive_cost_limited_astar_search"